src/main/kotlin/kotliquery/action/UpdateWithKeysQueryAction.kt

```diff
@@ -0,0 +1,11 @@
+package kotliquery.action
+
+import kotliquery.Query
+import kotliquery.Session
+
+data class UpdateWithKeysQueryAction(val query: Query) : QueryAction<Long?> {
```

Comment:
A bit verbose but `UpdateAndReturnGeneratedKeyQueryAction` is reasonable for me.